File: TornadoServer/model_one.py
import sys
import os
os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3'
os.environ["CUDA_VISIBLE_DEVICES"] = "-1"

import json 
import numpy  as np
from random import random,seed
from keras.models  import Sequential,load_model

seed(1)

class ModelOne():

    def __init__(self, config_file):
        
        with open(config_file) as json_file:
            data_config = json.load(json_file)
            model_path  = data_config["config"]["model_path"]
            self.__model   = load_model(model_path)        

    def predict(self, input):        
        np_input = np.array(input)
        prediction = self.__model.predict(np_input.reshape(1,8))        
        return prediction

def create_model_one(config_file):
    return  ModelOne(config_file)


if __name__ == '__main__':    
    model = ModelOne('config.json')
    prediction = model.predict([1,2,3,4,5,6,7,8])
    print(prediction)
    

File: PythonNet/ModelOne.cs
using Python.Runtime;
using System;
using System.Collections.Generic;

namespace PythonNet
{    
    public class ModelOne
    {
        private dynamic np;
        private dynamic load_models;
        private dynamic model;
        public ModelOne(string model_path)
        {
            PythonInstance.Execute( ()=>
            {
                dynamic os = Py.Import("os");
                os.environ["TF_CPP_MIN_LOG_LEVEL"] = new PyAnsiString("3");
                os.environ["CUDA_VISIBLE_DEVICES"] = new PyAnsiString( "-1");

                np = Py.Import("numpy");
                dynamic  keras_models = Py.Import("keras.models");
                load_models = keras_models.load_model;
                model = load_models(model_path);
            });
        }

        public double Predict(double v1, double v2, double v3, double v4, double v5, double v6, double v7, double v8)
        {
            var result = PythonInstance.Execute( ()=>
            {
                var input = new List<double>(new double[] { v1, v2,v3,v4,v5,v6,v7,v8});
                dynamic np_input = np.array(input);
                return model.predict(np_input.reshape(1,8));

            });
            return result[0][0];
        }
    }
}